cloud/aws/templates/aws_oidc/alarms.tf

```diff
@@ -0,0 +1,121 @@
+// CPU Utilization
+resource "aws_cloudwatch_metric_alarm" "cpu_utilization_too_high" {
+  count               = var.rds_create_high_cpu_alarm ? 1 : 0
+  alarm_name          = "rds-${data.aws_db_instance.civiform.id}-highCPUUtilization"
+  comparison_operator = "GreaterThanThreshold"
+  evaluation_periods  = var.rds_alarm_evaluation_period
+  metric_name         = "CPUUtilization"
+  namespace           = "AWS/RDS"
+  period              = var.rds_alarm_statistic_period
+  statistic           = "Average"
+  threshold           = var.rds_max_cpu_utilization_threshold
+  alarm_description   = "Average database CPU utilization is too high."
+
+  dimensions = {
+    DBInstanceIdentifier = data.aws_db_instance.civiform.id
+  }
+}
+
+resource "aws_cloudwatch_metric_alarm" "cpu_credit_balance_too_low" {
+  count               = var.rds_create_low_cpu_credit_alarm ? length(regexall("(t2|t3)", var.postgres_instance_class)) > 0 ? 1 : 0 : 0
+  alarm_name          = "rds-${data.aws_db_instance.civiform.id}-lowCPUCreditBalance"
+  comparison_operator = "LessThanThreshold"
+  evaluation_periods  = var.rds_alarm_evaluation_period
+  metric_name         = "CPUCreditBalance"
+  namespace           = "AWS/RDS"
+  period              = var.rds_alarm_statistic_period
+  statistic           = "Average"
+  threshold           = var.rds_low_cpu_credit_balance_threshold
+  alarm_description   = "Average database CPU credit balance is too low, a negative performance impact is imminent."
```

Comment:
**suggestion (blocking):** Is there a link we can include in the alarm description to help a naive incident responder understand what this means and how to respond to it?